modules/extensions-spring-support/src/test/java/org/mule/module/extension/internal/capability/xml/XmlModelEnricherTestCase.java

```diff
@@ -41,11 +43,12 @@
     @Test
     public void enrich()
     {
+        declarationDescriptor.named(EXTENSION_NAME).onVersion(EXTENSION_VERSION);
         XmlModelProperty xmlProperty = enrich(XmlSupport.class);
 
         assertThat(xmlProperty, is(notNullValue()));
         assertThat(xmlProperty.getSchemaVersion(), is(SCHEMA_VERSION));
```

Comment:
why is schema version different from the extension version in this case? per your code, the schema version defaults to the extension version